### PR TITLE
Add a bio for @ryanjames28

### PR DESCRIPTION
**Pull request by ghost**
so you know who I am. 

resolves issue -> #5 
@mollings, can you look at my :hankey:

### CODE REVIEW

**Comment by mollings** [comment]
What do you mean No

File: docs/ryanjames28.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+# ryanjames28
+
+**Name:** Ryan Bykowski
+
+**Occupation:** Software Engineer
+
+**Location:** Milwaukee Wisconsin
```
Comment (courtenayt) [inline docs/ryanjames28.md]: Hi @ryanjames28 I would suggest adding a comma between the city :beer: and state :cow2:

Comment (ghost) [inline docs/ryanjames28.md]: thanks i will do that right away

**Comment by CodeAlligator** [comment]
:congratulations:

**Comment by courtenayt** [comment]
:+1: looks good

**Pull request merged**